python/lsst/ts/cRIOpy/GUI/CustomLabels.py

```diff
@@ -322,35 +344,127 @@ def __init__(self, fmt="0.02f", is_warn_func=None, is_err_func=None):
+        super().__init__(signal, field, fmt, u.A)
+
+
+class Percent(DataUnitLabel):
     """Displays percents.
 
     Parameters
     ----------
+    signal : `Signal`, optional
+        When not None, given signal will be connected to method calling
+        setValue with a field from signal data. Field is the second argument.
+        Defaults to None.
+    field : `str`, optional
+        When specified (and signal parameter is provided), will use this field
+        as fieldname from data arriving with the signal. Defaults to None.
     fmt : `str`, optional
         Float formatting. Defaults to 0.02f.
     """
 
-    def __init__(self, fmt="0.02f"):
-        super().__init__(fmt, u.percent)
+    def __init__(self, signal=None, field=None, fmt="0.02f"):
+        super().__init__(signal, field, fmt, u.percent)
 
 
-class Volt(UnitLabel):
+class Volt(DataUnitLabel):
     """Displays Volts.
 
     Parameters
     ----------
+    signal : `Signal`, optional
+        When not None, given signal will be connected to method calling
+        setValue with a field from signal data. Field is the second argument.
+        Defaults to None.
+    field : `str`, optional
+        When specified (and signal parameter is provided), will use this field
+        as fieldname from data arriving with the signal. Defaults to None.
     fmt : `str`, optional
         Float formatting. Defaults to 0.02f.
     """
 
-    def __init__(self, fmt="0.02f"):
-        super().__init__(fmt, u.V)
+    def __init__(self, signal=None, field=None, fmt="0.02f"):
+        super().__init__(signal, field, fmt, u.V)
 
 
-class DataDegC(DataUnitLabel):
+class RPM(DataUnitLabel):
+    """Displays RPM.
+
+    Parameters
+    ----------
+    signal : `Signal`, optional
+        When not None, given signal will be connected to method calling
+        setValue with a field from signal data. Field is the second argument.
+        Defaults to None.
+    field : `str`, optional
+        When specified (and signal parameter is provided), will use this field
+        as fieldname from data arriving with the signal. Defaults to None.
+    fmt : `str`, optional
+        Float formatting. Defaults to .0f.
+    """
+
+    def __init__(self, signal=None, field=None, fmt=".0f"):
+        super().__init__(signal, field, fmt, u.Unit("min^-1"))
+
+
+class PressureBar(DataLabel):
+    """Display pressure in bar and psi"""
+
+    def __init__(self, signal=None, field=None):
+        super().__init__(signal, field)
+        self.unit_name = "bar"
+
+    def setValue(self, value):
+        psi = value * 14.5038
+        self.setText(f"{value:.04f} bar ({psi:.02f} psi)")
+
+
+class PressuremBar(DataLabel):
+    def __init__(self, signal=None, field=None):
+        super().__init__(signal, field)
+        self.unit_name = "mbar"  # this is only for display
+
+    def setValue(self, value):
+        value /= 1000.0
+        psi = value * 14.5038
```

Comment:
Maybe give the variable names to 1000.0 and 14.5038 for the readability?

Reply:
```python
        mbar = value * u.mbar
        bar = (mbar).to(u.bar).value
        psi = mbar.to(u.imperial.psi).value
```
is even nicer.